Formula/q/qt.rb

```diff
@@ -187,63 +183,73 @@ def install
       -archdatadir share/qt
       -datadir share/qt
       -examplesdir share/qt/examples
+      -hostdatadir share/qt/mkspecs
       -testsdir share/qt/tests
 
-      -no-feature-relocatable
+      -feature-webengine-kerberos
+      -pkg-config
       -system-harfbuzz
       -system-sqlite
+      -webengine-proprietary-codecs
 
+      -no-feature-clang
+      -no-feature-relocatable
       -no-sql-mysql
       -no-sql-odbc
       -no-sql-psql
     ]
 
-    cmake_args = std_cmake_args(install_prefix: HOMEBREW_PREFIX, find_framework: "FIRST") + %w[
-      -DFEATURE_pkg_config=ON
-      -DINSTALL_MKSPECSDIR=share/qt/mkspecs
-      -DQT_FEATURE_webengine_proprietary_codecs=ON
-      -DQT_FEATURE_webengine_kerberos=ON
-      -DQT_ALLOW_SYMLINK_IN_PATHS=ON
-    ]
+    cmake_args = %w[-DQT_ALLOW_SYMLINK_IN_PATHS=ON]
 
     if OS.mac?
-      # Fix a regression in Qt 6.5.2 w.r.t. system libpng
-      # https://bugreports.qt.io/browse/QTBUG-115357
-      cmake_args << "-DQT_FEATURE_webengine_system_libpng=OFF"
-
       cmake_args << "-DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}.0"
-      config_args << "-sysroot" << MacOS.sdk_path.to_s
+
+      config_args += %W[
+        -sysroot #{MacOS.sdk_path}
+        -no-feature-ffmpeg
+      ]
+      # FIXME: Workaround for error: use of undeclared identifier 'importMemoryHandleInfo'
+      # Remove once properly handled by Qt.
+      config_args << "-no-feature-webengine-vulkan"
       # NOTE: `chromium` should be built with the latest SDK because it uses
       # `___builtin_available` to ensure compatibility.
-      config_args << "-skip" << "qtwebengine" if DevelopmentTools.clang_build_version <= 1200
+      config_args += %w[-skip qtwebengine] if DevelopmentTools.clang_build_version <= 1200
     else
       # Explicitly specify QT_BUILD_INTERNALS_RELOCATABLE_INSTALL_PREFIX so
       # that cmake does not think $HOMEBREW_PREFIX/lib is the install prefix.
       cmake_args << "-DQT_BUILD_INTERNALS_RELOCATABLE_INSTALL_PREFIX=#{prefix}"
 
-      # The vendored copy of libjpeg is also used instead of the brewed copy, because the build
-      # fails due to a missing symbol otherwise.
-      # On macOS chromium will always use bundled copies and the QT_FEATURE_webengine_system_*
-      # arguments are ignored.
-      cmake_args += %w[
-        -DQT_FEATURE_webengine_system_alsa=ON
```

Comment:
Going to switch back to CMake cache variables as there is smart auto-ignore of configure args.